docs/usage/annotations.rst

```diff
@@ -144,3 +201,84 @@ To do so, one can declare an alias to the reusable qubits.
 .. note::
     Even if the reusable alias is not used in code, an alias must be defined to mark qubits as reusable.
     The identifier is insignificant.
+
+Entangled & Dirty Ancillae
+--------------------------
+If qubits are used within a snippet and are not annotated with either ``@leqo.output`` or ``@leqo.reusable``,
+they are treated by that snippet as entangled ancillae.
+Subsequent snippets interpret these qubits as dirty ancillae.
+Dirty ancillae may exist in an arbitrary quantum state, potentially entangled with other qubits,
+and must be explicitly annotated with ``@leqo.dirty`` to signify their intentional reuse in a snippet.
```

Comment:
maybe point out, that this is a reuse **as an input**, [for which the snippet consuming the dirty ancilla needs to opt in with the annotation]